### PR TITLE
Nmap import

### PR DESCRIPTION
**Pull request by MrJester**
New import module

### CODE REVIEW

File: modules/import/nmap.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                    self.insert_hosts(ip_address=ipaddress)
+
+            try:
+                for host_port in host.find('ports').findall('port'):
+                    if host_port.find('state').get('state') != 'open':
+                        continue
+                    port = host_port.attrib.get('portid')
+                    protocol = host_port.attrib.get('protocol')
+                    self.insert_ports(ip_address=ipaddress, port=port, protocol=protocol)
+            except AttributeError:
```
Comment (lanmaster53) [inline modules/import/nmap.py]: Probably excessive code inside the try/except again. Just wrap the `try` around the problematic line and `continue` in the except block to pass and keep processing. It's cleaner this way and will be easier to debug future errors, etc. Also, index. :-)

**Comment by MrJester** [comment]
Fixed this one.

File: modules.yml [hunk 2]
```diff
@@ -1020,3 +1020,13 @@
...
   required_keys: []
   version: '1.0'
+- author: Ryan Hays
+  dependencies: []
+  description: Imports hosts and ports into the respective databases from Nmap XML
+    output.
+  files: []
+  last_updated: '2019-05-28'
+  name: Nmap XML Output Importer
+  path: dev/nmap
```
Comment (lanmaster53) [inline modules.yml]: Same as other PRs.

File: modules.yml [hunk 3]
```diff
@@ -1020,3 +1020,13 @@
...
+  required_keys: []
+  version: '1.0'
```
Comment (lanmaster53) [inline modules.yml]: Same as other PRs.

Comment (MrJester) [reply modules.yml]: try it now...

**Pull request merged**